apps/codesandbox-react-next-template/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "codesandbox-react-next-template",
+  "description": "Template for creating codesandbox environment for @fluentui/react-next packages.",
+  "version": "7.0.0",
+  "private": true,
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "lint": "just-scripts lint",
+    "test": "just-scripts test",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "start": "just-scripts dev"
+  },
+  "devDependencies": {
+    "@types/react": "16.8.25",
+    "@types/react-dom": "16.8.4",
+    "@types/webpack-env": "1.15.1",
+    "@uifabric/tslint-rules": "^7.2.2",
+    "@uifabric/build": "^7.0.0",
+    "typescript": "3.7.2"
+  },
+  "dependencies": {
+    "@uifabric/set-version": "^7.0.15",
+    "@microsoft/load-themed-styles": "^1.10.26",
+    "@fluentui/react-next": "^8.0.0-alpha.56",
+    "@uifabric/theme-samples": "^7.0.103",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "tslib": "^1.10.0"
+  }
```

This can be simplified. (I'm guessing you don't need typescript as a devDependency, but if that's actually needed for the codesandbox environment you can bring it back.)
```suggestion
  "scripts": {
    "build": "just-scripts build",
    "lint": "just-scripts lint",
    "just": "just-scripts",
    "clean": "just-scripts clean",
    "code-style": "just-scripts code-style"
  },
  "devDependencies": {
    "@types/react": "16.8.25",
    "@types/react-dom": "16.8.4",
    "@uifabric/tslint-rules": "^7.2.2",
    "@uifabric/build": "^7.0.0"
  },
  "dependencies": {
    "@microsoft/load-themed-styles": "^1.10.26",
    "@fluentui/react-next": "^8.0.0-alpha.56",
    "@uifabric/theme-samples": "^7.0.103",
    "react": "16.8.6",
    "react-dom": "16.8.6",
    "tslib": "^1.10.0"
  }
```